src/pages/library/dataBrowser-utils.js

```diff
@@ -79,6 +84,11 @@ const normalizeDataset = dataset => {
       )(dataset['TerraDCAT_ap:hasDataUsePermission'])
     }
 
+  const access = isExternal(dataset) ?
+    datasetAccessTypes.EXTERNAL :
```

Comment:
I'm not wild about this since it now ignores the access type from the service. Maybe after some time we can go back and find a way to remove cases where the UI code looks at the `accessURL`.

Reply:
I agree. However, I think it already technically ignores access level for this field? Since the GRANTED/CONTROLLED is not technically part of the API.